BlockchainPatterns/bundles.rho

```diff
@@ -0,0 +1,32 @@
+new stdout(`rho:io:stdout`) in {
+
+  // Sending over a par-structured name works as expected
+  new a, b in {
+    @{*a | *b}!(Nil) |
+    for (_ <- @{*a | *b}) {stdout!("n")}
+  } |
+
+  // Order of parring doesn't matter as expected
+  new a, b in {
+    @{*a | *b}!(Nil) |
+    for (_ <- @{*b | *a}) {stdout!("o")}
+  } |
+
+  // Receiving on a bundled name works as expected
+  new chan in {
+    @{*chan}!(Nil) | // Also works as just `chan!(Nil) |`
+    for (_ <- @{bundle{*chan}}) {stdout!("s")}
+  } |
+
+  // But when par-structured and bundled meet... NOT as expected
```

Comment:
I don't see what's unexpected about this.

Reply:
Expected behavior: a comm event is produced and the letter prints to the console
Observed behavior: No letter prints to the console
Conclusion: No comm event produced

My expectations are based on Stay's description of the diffie-hellman-like pattern.